### PR TITLE
refactor(hosting.cron): use oui-action-menu

### PR DESCRIPTION
**Pull request by TedKac**

### CODE REVIEW

**Comment by TedKac** [comment]
Done 😄

File: client/app/hosting/cron/CRON.html [hunk 1]
```diff
@@ -120,14 +120,14 @@ <h2 data-translate="hosting_tab_menu_crons"></h2>
...
-                                            <span class="oui-icon oui-icon-trash_line oui-icon_small" aria-hidden="true"></span>
-                                        </button>
+                                        <oui-action-menu data-compact data-align="end">
+                                                <oui-action-menu-item data-on-click="modifyCron(cron)">
+                                                    <span data-translate="hosting_tab_CRON_edit_task"></span>
+                                                </oui-action-menu-item>
+                                                <oui-action-menu-item data-on-click="deleteCron(cron)">
+                                                    <span data-translate="hosting_tab_CRON_delete_task"></span>
+                                                </oui-action-menu-item>
+                                        </oui-action-menu>
```
Comment (JeremyDec) [inline client/app/hosting/cron/CRON.html]: Be careful with indentation :)

Comment (TedKac) [reply client/app/hosting/cron/CRON.html]: Fixed 😃

**Comment by TedKac** [comment]
Waiting for trad

**Pull request merged**